usr/share/rear/layout/save/TSM/default/400_check_tsm_configuration.sh

```diff
@@ -0,0 +1 @@
+CHECK_CONFIG_FILES+=( /etc/adsm/* )
```

Comment:
https://github.com/rear/rear/pull/2937/files
indicates that there is no newline at the end of the file